src/Fixer/Operator/AssignNullCoalescingToCoalesceEqualFixer.php

```diff
@@ -14,17 +14,15 @@
 
 namespace PhpCsFixer\Fixer\Operator;
 
-use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\AbstractShortOperatorFixer;
 use PhpCsFixer\FixerDefinition\FixerDefinition;
 use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
 use PhpCsFixer\FixerDefinition\VersionSpecification;
 use PhpCsFixer\FixerDefinition\VersionSpecificCodeSample;
-use PhpCsFixer\Tokenizer\Analyzer\RangeAnalyzer;
-use PhpCsFixer\Tokenizer\CT;
 use PhpCsFixer\Tokenizer\Token;
 use PhpCsFixer\Tokenizer\Tokens;
 
-final class AssignNullCoalescingToCoalesceEqualFixer extends AbstractFixer
+final class AssignNullCoalescingToCoalesceEqualFixer extends AbstractShortOperatorFixer
```

Comment:
Instead of using an abstract class, I would suggest deprecating this fixer and moving its logic to `LongToShorthandOperatorFixer`.

Reply:
Can we do this in a follow up PR? It will require making the new `LongToShorthandOperatorFixer` configurable so it can be used to only behave as the current `AssignNullCoalescingToCoalesceEqualFixer`.